docs/source/configure/index.rst

```diff
@@ -151,6 +151,8 @@ Environment-specific parameters
     }
   ]
 
+- ``max_past_created_time`` is an optional parameter specifying transaction how many hours in the past since current peer time transaction's `created_time` can be setted. The default value is `"24"` hours.
+
```

Comment:
I've changed the text as You suggested:
```
- ``max_past_created_time`` is an optional parameter specifying transaction how many hours in the past since current peer time transaction's `created_time` can be setted. The default value is `"24"` hours. The value should be the same in a whole iroha network.
```